Makefile.am

```diff
@@ -6,16 +6,24 @@ sshfs_SOURCES = sshfs.c cache.c cache.h
 if FUSE_OPT_COMPAT
 sshfs_SOURCES += compat/fuse_opt.c compat/fuse_opt.h
 endif
+if DARWIN_COMPAT
+sshfs_SOURCES += compat/darwin_compat.c compat/darwin_compat.h
+endif
 
 sshfs_LDADD = $(SSHFS_LIBS)
 sshfs_CFLAGS = $(SSHFS_CFLAGS)
 sshfs_CPPFLAGS = -D_REENTRANT -DFUSE_USE_VERSION=26 -DLIBDIR=\"$(libdir)\"
 
-EXTRA_DIST = sshnodelay.c
-CLEANFILES = sshnodelay.so
+EXTRA_DIST = sshnodelay.c FAQ.txt
```

Comment:
No, it didn't. I have removed this and ensured that `make distcheck` works (which required other fixes and seems to have been already broken in HEAD).